charts/mimir-distributed/templates/compactor/compactor-servmon.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.compactor.enabled -}}
```

Comment:
Compactor is a required component in Mimir. It shouldn't be disabled.

Reply:
artifact from starting from GEM chart I think, removed, will add to GEM changelog